content/projects/tdd/password-checker/part2/_index.md

```diff
@@ -2,20 +2,42 @@
 _db_id: 270
 content_type: project
 flavours:
-- any_language
+  - any_language
 from_repo: projects/tdd/password-checker/part1
 prerequisites:
   hard:
-  - projects/tdd/password-checker/part1
+    - projects/tdd/password-checker/part1
   soft: []
 ready: true
 story_points: 3
 submission_type: continue_repo
 tags:
-- logging
+  - logging
 title: Add logging to password checker
 ---
 
+## Set up environment
+
+### JavaScript:
+
+Your directory structure should look like this:
+
+```js
+├── spec
+|   ├── support
+|   |   └── jasmine.json
+|   └── password_is_valid_spec.js
+|   └── password_is_strength_spec.js
+├── src
+|   └── password_checker.js
+|   └── logger.js // new file for setting up your logger
+└── package.json
+```
+
+**Note**: Please export your both the previously created functions and the new `logger` setup object from their respective files
```

Comment:
Sentence structure.

Reply:
This sentence still doesn't make sense. Please fix the grammar. 

`Please export your both the previously created functions`. That is no good